tracer/test/Datadog.Trace.Security.IntegrationTests/Rcm/AspNetCore5AsmAttackerBlocking.cs

```diff
@@ -0,0 +1,139 @@
+// <copyright file="AspNetCore5AsmAttackerBlocking.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#if NETCOREAPP3_0_OR_GREATER
+
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Threading.Tasks;
+using Datadog.Trace.AppSec;
+using Datadog.Trace.AppSec.Rcm.Models.AsmFeatures;
+using Datadog.Trace.Configuration;
+using Datadog.Trace.TestHelpers;
+using Datadog.Trace.Vendors.Newtonsoft.Json.Linq;
+using FluentAssertions;
+using Xunit;
+using Xunit.Abstractions;
+using Action = Datadog.Trace.AppSec.Rcm.Models.Asm.Action;
+
+namespace Datadog.Trace.Security.IntegrationTests.Rcm;
+
+public class AspNetCore5AsmAttackerBlocking : RcmBase
+{
+    private const string AsmProduct = "ASM";
+
+    public AspNetCore5AsmAttackerBlocking(AspNetCoreTestFixture fixture, ITestOutputHelper outputHelper)
+        : base(fixture, outputHelper, enableSecurity: true, testName: nameof(AspNetCore5AsmAttackerBlocking))
+    {
+        SetEnvironmentVariable(ConfigurationKeys.DebugEnabled, "1");
```

Comment:
do we want debug enabled on master though ?

Reply:
We should not enable debug by default, as that's a different scenario to our customers, and it can hide issues

Reply:
Yes, I will remove that. I was merging the conflicting files before testing and setting the PR as ready to review, but I promise to remove the debug flag :)